logger.js

```diff
@@ -20,13 +20,16 @@ function pinoLogger (opts, stream) {
   var responseTimeKey = opts.customAttributeKeys.responseTime || 'responseTime'
   delete opts.customAttributeKeys
 
-  opts.serializers = opts.serializers || {}
-  var requestSerializer = opts.serializers[reqKey] || opts.serializers.req || serializers.req
-  var responseSerializer = opts.serializers[resKey] || opts.serializers.res || serializers.res
-  var errorSerializer = opts.serializers[errKey] || opts.serializers.err || serializers.err
-  opts.serializers[reqKey] = serializers.wrapRequestSerializer(requestSerializer)
-  opts.serializers[resKey] = serializers.wrapResponseSerializer(responseSerializer)
-  opts.serializers[errKey] = serializers.wrapErrorSerializer(errorSerializer)
+  opts.wrapSerializers = 'wrapSerializers' in opts ? opts.wrapSerializers : true
+  if (opts.wrapSerializers) {
+    opts.serializers = Object.assign({}, opts.serializers) || {}
```

Comment:
I'm cloning the serializers here as mentioned in the pinojs#95

Reply:
It would actually be better to clone the full object as well, it should not delete options.

Reply:
Do you mean the full ‘opts’ object?

Reply:
I've additionally cloned the `opts` object, so the `delete` operators won't cause issues.

Reply:
@mcollina I just tested this in my application and noticed `wrapSerializers` is output on the log itself! I'm guessing I need to temporarily remove it in `wrapChild` in the same way `opts.logger` and `opts.genReqId` is done?

Reply:
Yes, exactly